packages/wayland.rb

```diff
@@ -6,18 +6,20 @@ class Wayland < Package
   @_ver = '1.22.0'
   version @_ver
   license 'MIT'
-  compatibility 'x86_64 aarch64 armv7l'
+  compatibility 'all'
```

Comment:
So we haven't given up on i686 yet? :smile:

Reply:
It seems to build just fine, so I leave it.